src/Extend/Frontend.php

```diff
@@ -151,6 +159,10 @@ function (RouteCollection $collection, Container $container) {
                 /** @var RouteHandlerFactory $factory */
                 $factory = $container->make(RouteHandlerFactory::class);
 
+                foreach ($this->removedRoutes as $route) {
+                    $collection->removeRoute('GET', $route['name']);
```

Comment:
Why GET?

Reply:
because the routes are only added by `GET`, look right below this code

Reply:
Ah, gotcha